### PR TITLE
fix: modifications in code of ace/aci

### PR DESCRIPTION
**Pull request by Hardeep8587**
## Change Description

object creation in ace/aci manifest at the backend and design modifications 
ACI/ACE - notify parties array,third parties array, delivery destinations, various shipment type combinations
ACE search functionality

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Hardeep8587** [comment]
@kunalfleethawks  @amritfleethawks please review and merge! thanku !

**Review by kunalfleethawks — APPROVED** [review]

File: src/environments/environment.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ export const environment = {
   production: false,
 
 
-   BaseUrl: 'https://fleetservice.us-east-2.fleethawks.com/api/v1/',
```
Comment (kunalfleethawks) [inline src/environments/environment.ts:9]: Is this change required?

**Review by kunalfleethawks — APPROVED** [review]
@Hardeep8587 i see lot of code commented. If it’s not required. Please remove it.

**Review by amritfleethawks — APPROVED** [review]
